_sources/cice_3_user_guide.rst.txt

```diff
@@ -2262,7 +2262,80 @@ hemispheres, and must exceed a critical value nominally set to
 test and the Two-Stage test described in the previous section are
 provided in :cite:`Hunke2018`.
 
-
+In applying equations :eq:`t-distribution` through :eq:`short-means`,
```

Comment:
Just the lines I've highlighted are from my original write-up and have changed.  Matt has added everything after the "Practical Testing Procedure" header. My version of python was 2.7.10.

Reply:
Got it.